build.sh

```diff
@@ -31,10 +31,10 @@ SCRIPTPATH="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 DEBIANS="debian-stretch debian-buster debian-bullseye"
 UBUNTUS="ubuntu-bionic ubuntu-focal"
 CENTOSES="centos-7 centos-8"
-XPLAT_BASE="debian-buster"
-XPLAT_ARCHES="arm64v8 ppc64le"
+XPLAT_BASE="debian-bullseye"
+XPLAT_ARCHES="arm64 ppc64le"
```

Comment:
I might be able to revert this and change the architecture name in the underlying image, e.g. https://hub.docker.com/layers/apache/couchdbci-debian/bullseye-erlang-23/images/sha256-d72059db579e0497a44076c22ef3c32755ad49f0c2766c78f7550fff36e9fb4f?context=explore. But it's also not clear to me what the "right" answer is, e.g. I see lots of "arm64/v8" in Docker Hub. Regardless, it seems to be recognized as an ARM image just ine.